src/game.cpp

```diff
@@ -10872,24 +10872,45 @@ void game::chat()
     }
 
     uimenu nmenu;
-    nmenu.text = std::string( _("Who do you want to talk to?") );
-
+    nmenu.text = std::string( _( "Who do you want to talk to or yell?" ) );
```

Comment:
I haven't checked what the actual UI looks like, but you may not even need to add the "Yell" clause to this.